console/src/views/DisplaySettings.js

```diff
@@ -0,0 +1,102 @@
+import React from 'react';
+import kind from '@enact/core/kind';
+import {Panel} from '@enact/agate/Panels'
+import ColorPicker from '@enact/agate/ColorPicker';
+import {Row, Column, Cell} from '@enact/ui/Layout';
+import SliderButton from '@enact/agate/SliderButton';
+import Divider from '@enact/agate/Divider';
+import viewCss from './Settings.less';
+
+import AppContextConnect from '../App/AppContextConnect';
+
+const DisplaySettings = kind({
+	name: 'DisplaySettings',
+
+	styles: {
+		css: viewCss,
+		className: 'settingsView'
+	},
+
+	render: ({css, ...rest}) => (
+		<Panel {...rest}>
+			<Row className="enact-fit">
+				<Cell />
```

Comment:
What's with the empty cells? `<Cell />`
I get the impression the intent of this block is to position the content in the middle of the Panel-space. It seems a little complicated. Let's discuss how to simplify this section.

Reply:
Yeah honestly I just grabbed whatever was was in settings, but now would be a good time to fix both spots.